src/shims/unix/foreign_items.rs

```diff
@@ -394,13 +394,14 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             }
 
             // Querying system information
-            "sysconf" => {
+            "sysconf" | "__sysconf_xpg7" => {
```

Comment:
`__sysconf_xpg7` is not a standard name, is it? We should only support that on targets where it makes sense, i.e., Solaris.

Reply:
but neither is `__xpg_strerror_r` no ?

Reply:
Probably not, no -- we should do a similar check there. Just because we were sloppy in the past doesn't mean we should add more such things.